# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/test/mochitest.ini
# Commit: 9e9f7deff834
# Full Hash: 9e9f7deff8344b9f7dea8f09d302034ccda8bc7a
# Author: alwu <alwu@mozilla.com>
# Date: 2022-11-05 21:53:50
# Regressor Bug: 1262276
# File Overlap Count: 1
# Description:
#   Bug 1262276 - part14 : video-only seek should use original time as a seek target. r=padenot
#   
#   When requesting data from demuxer, it should always use normalized data,
#   which is within [0, original-track-duration]. So we need to adjust seek
#   target time in order not to use clock time which has grown over the
# ==============================================================================

diff -r 12aaeb776da1 -r 9e9f7deff834 dom/media/test/mochitest.ini
--- a/dom/media/test/mochitest.ini	Sat Nov 05 02:23:33 2022 +0000
+++ b/dom/media/test/mochitest.ini	Sat Nov 05 02:23:34 2022 +0000
@@ -812,6 +812,7 @@
 [test_seamless_looping_duration.html]
 [test_seamless_looping_cancel_looping_future_frames.html]
 [test_seamless_looping_media_element_state.html]
+[test_seamless_looping_resume_video_decoding.html]
 [test_seamless_looping_video.html]
 skip-if = toolkit == 'android' # Android has black frames issue.
 [test_video_to_canvas.html]